x-pack/filebeat/input/cel/input.go

```diff
@@ -529,10 +529,14 @@ func handleResponse(log *logp.Logger, state map[string]interface{}, limiter *rat
 					waitUntil = t
 				}
 			}
-			fallthrough
-		default:
-			delete(state, "events")
 			return false, waitUntil, nil
+		default:
+			status := http.StatusText(statusCode)
+			if status == "" {
+				status = "unknown status code"
+			}
+			state["events"] = map[string]interface{}{"error.message": fmt.Sprintf("failed http request with %s: %d", status, statusCode)}
```

Comment:
The `error.message` will result in a key containing a dot. Can you please change that to produce `{"error":{"message":...`.

Reply:
That should happen in the other `error.message` constructions too then.